apps/site/pages/en/about/index.mdx

```diff
@@ -77,3 +77,44 @@ by using our [`child_process.fork()`](https://nodejs.org/api/child_process.html)
 communicate with. Built upon that same interface is the [`cluster`](https://nodejs.org/api/cluster.html) module,
 which allows you to share sockets between processes to enable load balancing
 over your cores.
+
+## Official Node.js Resources
+
+To ensure authenticity and security when working with Node.js, always use official sources. Avoid trusting emails,
+binaries, or downloads from unofficial sources.
+
+### Official Node.js Domains
+
+For downloading Node.js binaries and accessing official documentation, use only these domains:
+
+- [nodejs.org](https://nodejs.org)
+- [nodejs.dev](https://nodejs.dev)
+- [iojs.org](https://iojs.org)
+
+### Official npm Packages
+
+The Node.js team maintains the following official npm package scopes:
+
```

Comment:
What about corepack, amaro, llhttp, etc...

Reply:
Ah yes, these do not have any @ within npm. Good catch. @marco-ippolito do you have a full list?

Reply:
I dont see amaro or undici in there 👀

Reply:
Oh, I guess those two weren't published from the foundation account. If you have a list of packages that belong to the organization, I'll check if they are published under the foundation account, and if not I'll add them to this list.

Reply:
cc @marco-ippolito